### PR TITLE
Attempt to start datastore from IDE

### PR DESCRIPTION
**Pull request by xulman**
Hi guys,

I wanted to be able to start an instance of DataStore server from IDE so that I could be testing this DataStore and potentially see better where it is failing... I couldn't really figure out the proper way of starting the application (DataStore) so I faked it by looking around to the unit tests to see how they start the server.... and that's what's inside this PR...

Could you, please, help me figure out a nicer way? is there any? could we polish this PR that way?

Thanks and cheers,
Vlado

### CODE REVIEW

File: src/test/java/cz/it4i/fiji/datastore/startDataStoreServer.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+import io.quarkus.test.junit.QuarkusTest;
+import org.junit.jupiter.api.Test;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+@QuarkusTest()
+public class startDataStoreServer {
```
Comment (kozusznik) [inline src/test/java/cz/it4i/fiji/datastore/startDataStoreServer.java]: Consider using UpperCase letter for class identifier.

**Review by kozusznik — APPROVED** [review]
Interesting approach👍

File: src/test/java/cz/it4i/fiji/datastore/startDataStoreServer.java (hunk 1 above)
Comment (xulman) [reply src/test/java/cz/it4i/fiji/datastore/startDataStoreServer.java]: sorry, fixed (how could I missed that...)

**Comment by xulman** [comment]
I wonder if there is a better way to start Quarkus/DataStore... especially, how to pass parameters to it in the source code -> I would like to avoid modifying the pom.xml, I think I'd be better to have the params directly in the test file... is there something like Java.setProperty?

**Comment by xulman** [comment]
(and thanks @kozusznik for reviewing this PR)

**Pull request merged**